### DISCUSSION TITLE
unable to access token from postman

### DISCUSSION THREAD

**Discussion by ajitk19** [egovernments/Digit-Core#50] [Local Governance]
All installations are done from https://core.digit.org/ and post-deployment I run the 

**sudo kubectl port-forward svc/egov-user 8080:8080 -n egov** 
output:- Forwarding from 127.0.0.1:8080 -> 8080
              Forwarding from [::1]:8080 -> 8080

After this, I run the access token on postman:

output in the console I got: {
                                               "error": "invalid_request",
                                               "error_description": "Invalid login credentials"
                                           }

data in postman: username - GRO
                            password - eGov@4321
                            tenantId - citya

**Comment by subhashini-egov** [comment]
Hi @ajitk19 you will have to create some users first prior to generating access token. Default users are not built-into the install yet. Please portforward the user service and run the _createnovalidate API to create users. It is part of the user collection.

**Reply by ajitk19** [reply]
Thanks for this...

Now, I'm not able to submit the complaints in the pgr section, which means the workflow isn't available.
Is there any way to create a workflow (business service)?

**Reply by subhashini-egov** [reply]
Yes. @ajitk19 - please check [this link](https://core.digit.org/platform/core-services/workflow-service) and see the "Reference Docs" section.

**Reply by ajitk19** [reply]
Hi Subhashini,

Thanks for your all help.

Recently I'm thinking about launching a production setup in my personal VM. Is it possible for all the production configurations to will work in my VM?

Can we configure all things in my S3 bucket?
